techniques/html/H73.html

```diff
@@ -1,5 +1,8 @@
 ---
-obsoleteMessage: The <code class="att">summary</code> attribute is obsolete as of HTML5.
+obsoleteMessage: |
+  The <code class="att">summary</code> attribute is
```

Comment:
> would it make sense to link directly to the obsolete feature? like this link for summary https://html.spec.whatwg.org/multipage/obsolete.html#attr-table-summary

Oh jeez, yes, I wanted to do that, and somehow didn't think the individual entries had IDs, which I was clearly wrong about. Thanks for pointing this out.

> i also wonder if we shouldn't just refer to these as being 'obsolete in html' rather than stating "html5" since the spec under the whatwg doesn't refer to itself that way (outside of the 'is this html5?' section where they acknowledge they will answer to that name, though it's more a buzzword term at this point

I think this goes hand-in-hand with Alastair's comment above, so I'll wait to hear back about that / will bring it up when I see Bruce.

Reply:
I am +1 to "obsolete since 2014 in HTML."